templates/cdk.master.auth-webhook.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=CDK master auth webhook
+After=network.target
+
+[Service]
+User=root
+WorkingDirectory=/root/cdk/auth-webhook
+ExecStart={{ charm_dir }}/../.venv/bin/gunicorn \
+    --bind {{ host }}:{{ port }} \
+    --capture-output \
+    --certfile /root/cdk/server.crt \
+    --disable-redirect-access-to-syslog \
+    --error-logfile auth-webhook.log \
```

Comment:
There seems to be a fair amount of periodic activity that ends up in the log file.  We probably need to set up log rotation and we might want to consider separating the error and access logs, or changing the default log level (should it be configurable somehow?).

Reply:
The debug level is set in the systemd service file.  We could add a config opt to k8s-master to control this, or document how users can modify that service file on-disk for all masters.  The former is nicer, but adds to our growing list of config opts.  Whatcha think?

Reply:
We've decided to keep debug hard coded (at least for now) and implement log rotation to address space issues.